packages/skills/carpark_client/skill.yaml

```diff
@@ -6,7 +6,8 @@ url: ""
 behaviours:
   - behaviour:
       class_name: MySearchBehaviour
-      args: {}
+      args:
+        period: 120
```

Comment:
This is too ambiguous a parameter, how about `ticker_period` or `tick_interval`?